app/controllers/api/v1x0/actions_controller.rb

```diff
@@ -33,6 +34,18 @@ def create
       def action_params
         params.permit(:operation, :processed_by, :comments)
       end
+
+      def rbac_scope(relation)
+        access_obj = RBAC::Access.new('actions', 'read').process
+        return relation if access_obj.admin?
+
+        raise Exceptions::NotAuthorizedError, "Not Authorized to list actions" unless access_obj.approver? || access_obj.accessible?
```

Comment:
Mustn't approver have accessible right? So check for `assessable?` should be sufficient.